robot/resources/keywords.resource

```diff
@@ -62,6 +62,7 @@ User adds ${title} book to catalog
     Execute Javascript    window.scrollTo(0, 800)
     Click Button    xpath://*[@aria-label='add']
     Wait Until Element Is Visible    (//h4[normalize-space()='Add book'])[1]
+    Input Text    xpath://input[@name='image']    asdasd
```

Comment:
If I understand correctly, there should be the image link here, since you're inputting the image link